build/Dockerfile

```diff
@@ -63,11 +63,11 @@ RUN apt-get update -qq && apt-get install -y -q --force-yes google-chrome-stable
 
 
 # Install nodejs, npm, gulp, etc
-RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
-RUN apt update -qq && apt install -y nodejs=6.12.3-1nodesource1
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
+RUN apt update -qq && apt install -y nodejs
```

Comment:
Would be good to pick a version for repeatability

Reply:
Done